app/controllers/creatives_controller.rb

```diff
@@ -79,7 +79,7 @@ def set_creative
 
   def creative_params
     params.require(:creative).permit(:creative_type, :name, :headline, :body, :cta).tap do |whitelisted|
-      whitelisted[:status] = params[:creative][:status] if authorized_user(true).can_admin_system?
+      whitelisted[:status] = params[:creative][:status] if authorized_user(true).can_admin_system? || !Current.organization.creative_approval_needed?
```

Comment:
This logic is getting more complex. I think it's time we wrap it in an `Authorizer`. What do you think?